src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -79,6 +80,19 @@ public override void OnActionExecuted(ActionExecutedContext context)
                         operationOutcomeResult.StatusCode = HttpStatusCode.ServiceUnavailable;
                         break;
                     case ResourceNotValidException _:
+                        if (context.ActionDescriptor is ControllerActionDescriptor)
+                        {
+                            var controllerDescriptor = (ControllerActionDescriptor)context.ActionDescriptor;
+                            if (controllerDescriptor.ControllerName == "Validate")
```

Comment:
Should probably use a `string.Equals` with a `StringComparison`

Reply:
Agreed, changed